extra/release_tool.py

```diff
@@ -2895,8 +2895,8 @@ def do_integration_versions_including(args):
 
 
 def find_repo_path(name, paths):
-    """ Try to find the git repo 'name' under some known paths.
-        Return abspath or None if not found.
+    """Try to find the git repo 'name' under some known paths.
+    Return abspath or None if not found.
```

Comment:
Here and below: these are unrelated changes and would be better to have them on a separate commit (facilitates cherry-picking)-